app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+
+  def update_posts_counter
+    user.increment!(:postsCounter)
+  end
```

Comment:
## Optional
- [ ] I suggest that you use one of the Rails Active Callbacks - `after_save` to call the `#update_posts_counter` method every time a new post has been saved to the database, then make the `update_posts_counter` method private